gradle.properties

```diff
@@ -27,3 +27,6 @@ version=2.20.0-SNAPSHOT
 sdk_version=2.20.0.dev
 
 javaVersion=1.8
+
+docker_image_default_repo_root=apache
+docker_image_default_repo_prefix=beam-
```

Comment:
This is intended in order to distinguish project name and image name.

Reply:
Ideally we would be able to use a subrepo or a slash in the image name, but since those are not supported by Docker, I don't think mixing hyphens and underscores is a good alternative. I think most users would expect either only hyphens or only underscores (the two are seldom mixed within the same string), so having both makes tags more difficult to use for humans (harder to remember and easy to make typos).

Reply:
changed it to underscore.

Reply:
Thanks! Make sure you also change the prefix everywhere it needs to be hard-coded.